plugins/lookup/aws_service_ip_ranges.py

```diff
@@ -55,9 +57,16 @@
 
 class LookupModule(LookupBase):
     def run(self, terms, variables, **kwargs):
+        if "ipv6_prefix" in kwargs["ipv6_prefix"] is False:
+            prefixes_label = "prefixes"
+            ip_prefix_label = "ip_prefix"
+        else:
+            prefixes_label = "ipv6_prefixes"
+            ip_prefix_label = "ipv6_prefix"
```

Comment:
The logic here looks a little broken.  I'd suggest tweaking it a little:

```suggestion
        if "ipv6_prefix" in kwargs and kwargs["ipv6_prefix"]:
            prefixes_label = "ipv6_prefixes"
            ip_prefix_label = "ipv6_prefix"
        else:
            prefixes_label = "prefixes"
            ip_prefix_label = "ip_prefix"
```